COLLABORATOR_GUIDE.md

```diff
@@ -87,6 +86,18 @@ The TC should serve as the final arbiter where required.
 
 ## Landing Pull Requests
 
+Pull requests should be landed with Jenkins as documented [here]
+(https://github.com/nodejs/node/wiki/Merging-pull-requests-with-Jenkins). 
+
+#### Landing Pull Requests manually (for emergency only)
+Pull requests should be landed with Jenkins as documented [here]
```

Comment:
What if we changed the language just a little bit so it doesn't look like a copy paste job went wrong?